### DISCUSSION TITLE
Aqara/xiaomi compatible routers/repeaters

### DISCUSSION THREAD

**Discussion by pedroke** [Koenkk/zigbee2mqtt#12237] [Devices]
Hi guys, 
since Aqara devices are so popular and widely used and as we know they are not compatible with many routing devices, I would like to start here a discussion about your experience.
So far I found only this older discussion 
https://community.hubitat.com/t/xiaomi-aqara-devices-pairing-keeping-them-connected/623/896
To summarize, the discussion recommends as compatible:

IKEA TRÅDFRI LED Bulbs
TRÅDFRI Signal repeater
TRÅDFRI Wireless control outlet
Nue / 3A Home
Twin Power Point GPO Switch 103 - model MG-AUWSR01
OSRAM/Sylvania LIGHTIFY
A19 RGBW color recessed light 
RT Tunable White recessed light
OSRAM Smart+ Outlet - model AB3257001NJ (UK & EU only)
Sylvania SMART+ Plug 72922 (mostly working, more info here & here, but the 72922-A variant does not work)
Sinopee Thermostat TH1124ZB
XBee Zigbee Series 2 / Series 3 modules

Since the list is quite outdated (2018) it would be interesting to know the current experience.
I would be especially interested in Blitzwolf plugs.. 
Do you guys have any experience?

**Comment by pedroke** [comment]
I just got a confirmation from the BW SHP-13 owner that it works fine as a router for Xiaomi sensors.. So I ordered one and I'll see.. I will get back here to confirm.

**Comment by groto27** [comment]
I got a few IKEA plugs this week and I can't get my Aqara Cube or Motion Sensor to connect to them. I can only get those devices to connect to the Coordinator. I thought the plugs were supposed to be one of the few devices that would work with Aqara as routers.

Before this I tried Sonoff S31 Zigbee plugs and they don't connect to those either. I am not sure what to do.

**Reply by pedroke** [reply]
Usually when there is a problem with routing Xiaomi devices, they join the network to router with no problems, but they stop working after some rather short time.. At least this is my experience.. I'm not sure what to do if they cannot join to router.. I suppose you have the latest coordinator firmware and z2m. I can remember there were similar problems in some versions, but I'm not sure..

**Comment by lux73** [comment]
Hue white ambiance E26/E27 - [8718696548738](https://www.zigbee2mqtt.io/devices/8718696548738.html#philips-8718696548738)
Hue white ambiance E14 - [8718696695203](https://www.zigbee2mqtt.io/devices/8718696695203.html#philips-8718696695203)
Hue white ambiance GU10 - [8718696598283](https://www.zigbee2mqtt.io/devices/8718696598283.html#philips-8718696598283)
Dresden Elektronik -[XVV-Mega23M12](https://www.zigbee2mqtt.io/devices/XVV-Mega23M12.html#dresden%2520elektronik-xvv-mega23m12)
Dresden Elektronik - [Mega23M12](https://www.zigbee2mqtt.io/devices/Mega23M12.html#dresden%2520elektronik-mega23m12)

all the postetd above works without any Glitches with my 25 Xiaomi Devices

**Reply by codyc1515** [reply]
Aqara devices work okay connected directly to Hue devices but they don't re-route to another Hue device if one of the Hue devices go down.

**Comment by KHV8** [comment]
Using a sonoff 3.0 usb dongle as coordinator, All the below mains work with aqara battery devices

frient smart cable
aqara single switch t1
aqara EU plug
Tube router
Different types of Hue bulbs
Hue plugs

**Comment by pedroke** [comment]
Btw I've got few of Immax Neo 16A plugs https://www.zigbee2mqtt.io/devices/07048L.html and they do NOT work correctly with Aqara battery powered sensors.

**Comment by pedroke** [comment]
I did some investigation and found that such sleepy end devices (e.g. Aqara battery sensors) which do not implement Zigbee 3.0 keep alive mechanism correctly would probably never work with up to date Zigbee 3.0 certified routers which implement the aging mechanism as defined in specification. So in such case we are probably left with older or custom devices.

**Comment by pedroke** [comment]
Now I can confirm from my own experience that Blitzwolf BW SHP-13 routes fine Xiaomi/Aqara sensors. So it is one of very few 16A sockets with power metering which works fine with Aqara..

**Comment by nprez83** [comment]
I'm currently using the new Sonoff S40 lyte with Z2M and my aqara devices (RTCGQ11LM motion and MCCGQ11LM contact sensors) are connecting well to these, stable, no drop offs, would recommend. A few tips. For the aqara devices that I want to connect via a repeater I make sure to Permit Join via that specific device. The S40s generally recognize aqara devices quickly, but not without occasional quirks. For example, sometimes on the first join the battery percentage is not recognized, the fix for that is let it finish the interview and then put the aqara device back into pairing mode (hold the button till it flashes 3 times) while the Permit join is still active, this usually makes the S40 reassign a network ID to the device and fixes the battery percentage 9 times out of 10. If it doesn't, force remove the device and try again.

I also tried the Thirdreality plugs 3RSP019BZ, and those did not work well at all. I had a really hard time getting these to recognize the aqara devices, although I did manage to pair a couple of motion sensors and one contact sensor after a bunch of tries, definitely not as reliable as the S40s. That sucks cuz I really wanted these to work as they are slightly smaller and sleeker than the S40s, but oh well, back to Amazon they go.

Lastly, I also use Innr bulbs (AE 260), which are not really advertised as repeaters in Amazon but Z2M recognizes them as such so I figured I'd try. The aqara devices do connect to them easily and without much issues. Unfortunately, the devices drop off within the hour, so something is off, unclear to me whether it's on the aqara or Innr side, or more likely a combination of both. I've read [some Zigbee lights only serve as repeaters for other lights](http://thingsthataresmart.wiki/index.php?title=Is_this_device_a_repeater%3F) so that may be what's happening. Oh well, they're decent lights so I'm keeping them.

**Comment by oystermon** [comment]
Copied from another post I made:

 I had a xiaomi motion sensor that frequently disconnected and this is what helped me:

- Having a strong mesh.. I bought some IKEA smart plugs and placed them around my home to do that. I also add smart bulbs and that seemed to work

- Move your zigbee coordinator away from your machine via a USB 2.0 extension cable (to reduce interference) see https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html#usb-based-adapter

- When pairing the motion sensor try to pair it to the closest compatible router instead (in my case it was an IKEA plug) of the permit all, and pair it in the final location of where you are going to place the motion sensor.

**Comment by luisiam** [comment]
I came across this discussion while searching on Google. I wanted to add that newer Third Reality plug 3RSP02028BZ is able to recognize my Xiaomi motion sensor and contact sensor without any issue. I am using ZHA though. Since I added all my battery sensors before the smart plug, all the battery sensors is connecting to the coordinator directly. Should I repair them to the smart plug router instead?

**Comment by brianacox** [comment]
I brought a M2 a few weeks ago and found that it works ok with my legacy sensors, but great with the newer stuff..... BUT distance is still an issue so I got a couple of T1 LED strips (which should act as repeaters) but despite engaging Aqara support can't seem to establish how to make them repeat... or if they do, by default, then they don't appear to be doing a good job (I have another Zigbee network, based on Conbee which works really well with wired repeaters).  I was wondering if I should by another wired devices - any suggestion what - and see if that acts as a better repeater.  Clearly it needs to be something compatible with an M2.

**Reply by nprez83** [reply]
I've actually been using these cheap [Tuya zigbee presence sensors](https://a.aliexpress.com/_mNGrku2) from Aliexpress and they work really well as repeaters for aqara sensors, both the new and older version.

**Reply by brianacox** [reply]
Great… I will take a look…. Presumably they work with an M2; it’s seems very fussy about what it supports.

**Reply by brianacox** [reply]
P.S. how do you know they are acting as repeaters?   (excuse the silly question but I don't think my T1 Led strips (which are sold by Aqara as repeaters/mesh extenders etc) don't seem to be doing that).   Aqara support asked me to deploy them, then press the reset button on the door/window sensors and temp sensors three times and they would then join the new mesh.....I tried that (I must admit only once as I wasn't convinced by the reply) and didn't see any improvement.

**Reply by nprez83** [reply]
Z2M tells you which devices are routers vs end points. These presence sensors are detected as routers. Additionally, the Z2M map shows you your entire network, so I know several of my aqara devices are connected to these presence sensors rather than the coordinator.

**Reply by brianacox** [reply]
Ok that makes sense …. I was on Z2M before.   Shame that the Aqara hub doesn’t present the same network diagram.

**Comment by Tahutipai** [comment]
Tip for anyone having problems pairing Aquara battery sensors (of various types): some of the devices seem to "go to sleep" during the pairing process. Especially motion sensors. Solution: after the device is in pairing mode, and you have started the pairing process in HA: continue to press the pairing mode button, just once ever 3 seconds or so. 

Quick pushes, but not frequent pushes.

To be clear: you are just pushing then immediately releasing the button -- not holding it down. I.e the button will be in the depressed state (😓) for about just two tenths of a second, every 3 seconds or so.

This tip increased my pairing success rates to 100%

**Reply by pedroke** [reply]
Thanks for the nice tip... Unfortunately the problem with some Xiaomi/Aqara devices is that they don't even expose the necessary cluster for binding, e.g. genOnOff only the genBasic, so the binding is not even possible when the device is kept alive :(

**Comment by mahood73** [comment]
I have four of these, and my Aquara devices (climate sensors, push buttons and water leak sensors) all happily pair to them and work just fine.
https://www.zigbee2mqtt.io/devices/MOES_plug.html

**Reply by rahulpdev** [reply]
> I have four of these, and my Aquara devices (climate sensors, push buttons and water leak sensors) all happily pair to them and work just fine. https://www.zigbee2mqtt.io/devices/MOES_plug.html

Are your Moes devices the same as these, they look very similar? https://www.aliexpress.com/item/1005006346571151.html
What model number shows up for you in Home Assistant?

**Comment by mahood73** [comment]
I believe so, probably the same thing with a different branding.

I see this in HA:
```
Zigbee Model
TS011F
Zigbee Manufacturer
_TZ3000_yujkchbz
```

**Comment by lachlanmacphee** [comment]
Thought I'd add to this thread since it pops up on Google. The new IKEA TRETAKT smart plugs are NOT compatible with Aqara T1 sensors in my (and others') experience:

https://community.home-assistant.io/t/ikea-tretakt-plugs-as-zigbee-routers-for-aqara-sensors/766857/5
